training/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# pull base image
+FROM condaforge/miniforge-pypy3
+
+WORKDIR /usr/src/training
+
+# set environment variables
+
+# install dependencies
+RUN mamba create --name dlp -y
+
+COPY environment.yml pyproject.toml ./
+RUN mamba run --live-stream -n dlp mamba env update --file environment.yml --prune
+RUN mamba run --live-stream -n dlp poetry install
```

Comment:
Minor thing, but add the `--sync` flag so that it installs from mamba, and add `--without dev` flag so that it doesn't install dev dependencies in prod (`poetry install --without dev --sync`). Bonus points if you can get it working so that it adds the `--without dev` flag only during docker prod build.